torch/_inductor/triton_ops/autotune.py

```diff
@@ -575,6 +575,12 @@ def get_grid_dim(numel, block_name, block):
                 f"TritonKernel.indexing assumes {label.lower()}numel == 1 => {block_name} == 1"
                 f"({label.lower()}numel=={numel}, {block_name}={block})."
             )
+        max_block = config.triton.max_block[label]
```

Comment:
Suggest moving this assert to where we construct configs and autotune them as a minor perf optimization (that code runs just once).